src/styles/index.scss

```diff
@@ -1,3 +1,130 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+.card {
+  width: 166px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+
+  &__img {
+    display: flex;
+    flex-direction: column;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__text {
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+    color: #060b35;
+    width: 100%;
+    height: 36px;
+    margin-bottom: 0;
+  }
+
+  &__code {
+    line-height: 14px;
+    font-size: 10px;
+    font-weight: 400;
+    color: #616070;
+    width: 100%;
+    height: 14px;
+    text-align: left;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 166px;
+    margin-bottom: 16px;
+
+    &-text {
+      display: flex;
+      line-height: 18px;
+      font-size: 12px;
+      font-weight: 400;
+      color: #616070;
+    }
+
+    &-value {
+      display: flex;
+      line-height: 18px;
+      font-size: 16px;
+      font-weight: 700;
+      color: #060b35;
+    }
+  }
+
+  &__link {
+    width: 166px;
+
+    &-button {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      text-transform: uppercase;
+      text-decoration: none;
+      width: 166px;
+      height: 40px;
+      font-weight: 700;
+      border-radius: 5px;
+      background-color: #00acdc;
+      color: #fff;
+    }
+
+    &-button:hover {
+      border: 1px solid #00acdc;
+      background-color: #fff;
+      color: #00acdc;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  width: 166px;
+  margin-bottom: 24px;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+    border-radius: 0.5px;
+  }
+
+  &__star:nth-child(-n + 4) {
```

Comment:
The selector `&__star:nth-child(-n + 4)` might not be clear in terms of its intent. Consider adding a comment to explain that it targets the first four child elements, or use a more descriptive class name if possible.